iceoryx_binding_c/source/cpp2c_enum_translation.cpp

```diff
@@ -100,5 +100,25 @@ iox_ListenerResult ListenerResult(const iox::popo::ListenerError value)
     }
 }
 
+iox_SubscriberTooSlowPolicy subscriberTooSlowPolicy(const iox::popo::SubscriberTooSlowPolicy policy)
+{
+    switch (policy)
+    {
+    case SubscriberTooSlowPolicy::WAIT_FOR_SUBSCRIBER:
+        return SubscriberTooSlowPolicy_WAIT_FOR_SUBSCRIBER;
+    default:
+        return SubscriberTooSlowPolicy_DISCARD_OLDEST_DATA;
```

Comment:
Rejected! This makes no sense here since this enum is never used for an error case or in combination with an expected. Actually, it replaces a bool where we decided to use an enum for better readability.

Reply:
Ok, you're right. But what happens if we choose to extend this enum for some weird reason?